### PR TITLE
Add integration with TravisCI

### PR DESCRIPTION
**Pull request by wack17s**
- Add TravisCI badge (master branch) to README.md
- Add .travis.yml

### CODE REVIEW

**Comment by wack17s** [comment]
Closes #4

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: python
+python: 3.6
+cache: pip
+before_install: pip install -U pre-commit
+script: pre-commit run --all-files
+notifications:
+  email: false
```
Comment (webknjaz) [inline .travis.yml]: don't do this.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
```
Comment (webknjaz) [inline .travis.yml]: plz have normal EOL here

**Review by webknjaz — DISMISSED** [review]
+Rebase this branch on top of master instead of merging master into it

**Comment by wack17s** [comment]
@webknjaz have i to pull master before merge or i can just rebase it again?

**Comment by webknjaz** [comment]
You have to master your English :)

**Comment by webknjaz** [comment]
Rebase on top of master

**Comment by webknjaz** [comment]
And make treats pass before merging

**Review by webknjaz — APPROVED** [review]

**Pull request merged**